terraform/hcl/applicable/req_sw_terraform_aws_api_gateway_auth.tf

```diff
@@ -0,0 +1,6 @@
+resource "aws_api_gateway_method" "vulnerable_method" {
+  rest_api_id   = "dummy"
+  resource_id   = "dummy"
+  http_method   = "GET"
+  authorization = "NONE"
+}
```

Comment:
[comment]: <> (FrogbotReviewComment)


## 🛠️ Infrastructure as Code Vulnerability
<div align='center'>

| Severity                | Finding                  |
| :---------------------: | :-----------------------------------: |
| ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    High | authorization=NONE was detected |

</div>

<details>
<summary> <b>Full description</b> </summary>
<br>

All `aws_api_gateway_method` resources should have `authorization != NONE`

Vulnerable example -
```
resource "aws_api_gateway_method" "vulnerable_example" {
  authorization = "NONE"
}
```

Secure example -
```
resource "aws_api_gateway_method" "secure_example" {
  authorization = "AWS_IAM"
}
```

</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://github.com/jfrog/frogbot#readme)

</div>